pages/api/contacts/index.js

```diff
@@ -0,0 +1,23 @@
+import dbConnect from "../../../db/connect";
+import Contact from "../../../db/models/contact";
+
+export default async function handler(request, response) {
+  await dbConnect();
+
+  if (request.method === "POST") {
+    try {
+      const contactData = request.body;
+      await Contact.create(contactData);
+      response.status(201).json({ message: "Contact created" });
+    } catch (error) {
+      console.log(error);
+      response.status(400).json({ error: error.messge });
+    }
+  }
+
+  if (request.method === "GET") {
```

Comment:
To conform to a correct error handling you should also add a `try / catch` statement to your get request block.